hazelcast/src/main/java/com/hazelcast/map/merge/IgnoreMergingEntryMapMergePolicy.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.merge;
+
+import com.hazelcast.core.EntryView;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+
+import java.io.IOException;
+
+/**
+ * IgnoreMergingEntryMapMergePolicy causes the merging entry to be ignored while collecting merge-needed entries on merging side
+ * Its operations are not implemented since it is only used as a marker
+ * @since 3.9
+ */
+public class IgnoreMergingEntryMapMergePolicy implements MapMergePolicy {
```

Comment:
I would go with `final` here. it's a very specialized policy and you do not want users to subclass it.